code/modules/clothing/spacesuits/rig/modules/combat.dm

```diff
@@ -113,47 +113,109 @@
 
 /obj/item/rig_module/mounted
 	name = "mounted laser cannon"
-	desc = "A shoulder-mounted battery-powered laser cannon mount."
+	desc = "Some sort of mounted gun."
 	selectable = TRUE
 	usable = TRUE
 	module_cooldown = 0
 	icon_state = "lcannon"
 
 	engage_string = "Configure"
 
-	category = MODULE_HEAVY_COMBAT
+	category = MODULE_LIGHT_COMBAT
```

Comment:
I think the re-classifies a lot of modules as light combat.

I think we should add a ballistic category (as the Primary balance there is the ammo usage) and update the combat category names to indicate that those are laser weapons.

(Maybe even a ballistic heavy / ballistic light/ launcher category)